File: stack/1598. Crawler Log Folder/main.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        int num = 0;
        for(auto& log : logs){
            if(log == "./"){
                continue;
            }
            else if(log == "../"){
                if(num > 0){
                    num = num -1;
                }
            }
            else{
                num = num + 1;
            }
        }
        return num;
    }
};

File: stack/1598. Crawler Log Folder/main.py
class Solution(object):
    def minOperations(self, logs):
        """
        :type logs: List[str]
        :rtype: int
        """
        num = 0
        for log in logs:
            if log == "./":
                continue
            elif log == "../":
                if num > 0:
                    num = num - 1
            else:
                num = num+1
        return num
